### DISCUSSION TITLE
TOTP registration QR-Code won't be generated.

### DISCUSSION THREAD

**Discussion by feuerstein-dev** [authelia/authelia#4876] [Q&A] [answered]
Hello, today i reinstalled Authelia on my machine. Everything is working, except the QR Code for TOTP won't be valid. I tried resetting the config file a few times, or regenerating the docker-compose stack, after deleting the database (I'm using the standard sqlite3 option). I did the same procedure as last time i installed authelia, but it doesn't work. Everything is behind my reverse proxy (NginX Proxy Manager), but i did not changed anything here, so this should be fine i guess. Here are my logs:

`authelia_1  | time="2023-02-03T20:45:00+01:00" level=error msg="Cannot parse token" method=POST path=/api/secondfactor/totp/identity/finish remote_ip=<my-ip> stack="github.com/authelia/authelia/v4/internal/middlewares/authelia_context.go:65       (*AutheliaCtx).Error\ngithub.com/authelia/authelia/v4/internal/middlewares/identity_verification.go:144 IdentityVerificationFinish.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/require_first_factor.go:15   Require1FA.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/bridge.go:54                 (*BridgeBuilder).Build.func1.1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:25                SecurityHeadersCSPNone.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:35                SecurityHeadersNoStore.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:16                SecurityHeaders.func1\ngithub.com/fasthttp/router@v1.4.14/router.go:414                                  (*Router).Handler\ngithub.com/valyala/fasthttp@v1.43.0/http.go:154                                   (*Response).StatusCode\ngithub.com/valyala/fasthttp@v1.43.0/server.go:2338                                (*Server).serveConn\ngithub.com/valyala/fasthttp@v1.43.0/workerpool.go:224 `

Thanks in advance :D

**Comment by james-d-elliott** [comment, marked as answer]
If you clicked or copied the link your mail server or mail client is mangling the link. Alternatively if you copied it by highlighting it in the text version then you didn't copy the full link most likely.

**Reply by feuerstein-dev** [reply]
Seems like i did not copy the full link out of the terminal. I tried it with a text editor right now and its working seamlessly.  Thank you :D

**Reply by james-d-elliott** [reply]
No worries!
